minibot/utils.py

```diff
@@ -0,0 +1,21 @@
+"""
+Utils for minibot.
+"""
+
+class Utils:
```

Comment:
Not really a fan of this file. `distance` is called only once, `normalize` is called twice.

Reply:
Done.